.github/workflows/csfix.yml

```diff
@@ -0,0 +1,21 @@
+name: Format PHP
+
+on: pull_request
+
+jobs:
+  php-cs-fixer:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+
+    - name: Run php-cs-fixer
+      uses: docker://oskarstark/php-cs-fixer-ga
+
+    - uses: stefanzweifel/git-auto-commit-action@v3.0.0
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I have not yet tried providing a secret with the name `GITHUB_TOKEN`, so I can't tell whether that one will take precedence over the `GITHUB_TOKEN` that is automatically provided - perhaps it makes more sense to create a differently named token?

Reply:
Cool, looks like it worked: 25fefe481c142e16c74ce6fbc1331e3e0221c335!

Reply:
I guess it didn't worked because the automated PR did not trigger new builds

Reply:
Ha, true!

Perhaps provide the secret as `STAAB_TOKEN`?

😉

Reply:
yep, will provide a token ;). thx

Reply:
Let me know if I should adjust #23 accordingly!